packages/@okta/vuepress-site/docs/guides/configure-ciba/main/index.md

```diff
@@ -97,24 +97,22 @@ There are a few steps required to configure CIBA for your org:
 
 ### Create an OIDC client integration and enable CIBA
 
-Create an Okta OIDC client app integration to represent the consumption device, enable CIBA as a grant type, and associate the client app with the [custom authenticator that you previously configured using the Devices SDK](https://github.com/okta/okta-mobile-swift).
+Create an Okta OIDC client app integration to represent the consumption device. Then, enable CIBA as a grant type and associate the client app with the [Custom Authenticator that you previously configured using the Devices SDK](https://github.com/okta/okta-mobile-swift).
 
 > **Note:** Web apps are currently the only application supported with the CIBA flow.
 
-1. In the Admin Console, go to **Applications** > **Applications**.
-2. Click **Create App Integration**.
-3. Select **OIDC - OpenID Connect** as the **Sign-in method** and choose which type of application that you want to integrate with Okta. In this example, select **Web Application**.
-4. Click **Next**.
-5. Enter a name for your app integration.
-6. Select the **Client-initiated backchannel authentication flow (CIBA)** checkbox as a **Grant type**.
-7. Select the custom authenticator that you [previously configured using the Mobile SDK](https://github.com/okta/okta-mobile-swift) from the **Preferred authenticator for CIBA** dropdown list.
-8. In the **Sign-in redirect URIs** box, specify the callback location where Okta returns the tokens after the user finishes authenticating. You can use the default URI for this exercise.
+1. In the Admin Console, go to **Applications** > **Applications**, and then click **Create App Integration**.
+2. Select **OIDC - OpenID Connect** as the **Sign-in method** and choose which type of application that you want to integrate with Okta. In this example, select **Web Application**.
+3. Click **Next**, and then enter a name for your app integration.
+4. Select the **Client-initiated backchannel authentication flow (CIBA)** checkbox as a **Grant type**.
```

Comment:
Could simplify to:

Select **Client-initiated backchannel authentication flow (CIBA)** as a **Grant type**.